test_conformance/math_brute_force/main.c

```diff
@@ -657,10 +596,12 @@ static void PrintUsage( void )
     vlog( "\n" );
 }
 
+#if 0
```

Comment:
Why is this commented out? Should it be deleted?

Reply:
Yes this needs to be deleted